### PR TITLE
Change to png asciinema preview

### PR DESCRIPTION
**Pull request by untitaker**
The SVG version locks up the tab for me in Firefox, and I am on a fairly beefy machine. The preview image is also very large, so downscaling it a bit may be preferrable for saving vertical space + help with sharpness on retina

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ A common use case could be the following:
 
 `suckit http://books.toscrape.com -j 8 -o /path/to/downloaded/pages/`
 
-[![asciicast](https://asciinema.org/a/327889.svg)](https://asciinema.org/a/327889)
+<a href="https://asciinema.org/a/327889"><img alt="asciicast" src="https://asciinema.org/a/327889.png" width="500" /></a>
```
Comment (Skallwar) [inline README.md:41]:
```suggestion
[![asciicast](https://asciinema.org/a/327889.png)](https://asciinema.org/a/327889)
```

Is this not working ?

Comment (untitaker) [reply README.md:41]: yes but I also wanted to limit image size so I have to scroll less. your call though

Comment (Skallwar) [reply README.md:41]: Ok. Can you center it ?

**Review by CohenArthur — APPROVED** [review]

**Comment by Skallwar** [comment]
For some reason the play button is cropped. That's weird. And if people don't understand that you need to click on it, I don't know if it is a good idea...

**Comment by untitaker** [comment]
idk how to fix the play button, I think I'll just give up on this

**Comment by Skallwar** [comment]
Thanks anyway

**Comment by pinkforest** [comment]
Hey @untitaker mind having a re-look now?

I went down the rabbithole with #139

**Comment by untitaker** [comment]
lgtm, thank you!

**Comment by Skallwar** [comment]
Tanks for your work, both of you

**Pull request closed without merging**